src/server.jl

```diff
@@ -95,29 +92,66 @@ end
 
 Return the filesystem path corresponding to a requested path, or an empty
 String if the file was not found.
+
+Cases:
+
+* an explicit pointer with an `index.html` (e.g. `foo/bar/index.html`) is
+given --> change WEB_DIR and serve the page
+* an implicit pointer to an `index.html` (e.g. `foo/bar/` or `foo/bar`) is
+given --> change WEB_DIR and serve the page
+* an explicit pointer to a file is given (e.g. `/sample.jpeg`) --> a full
+path is constructed and served
+* an explicit pointer to a dir without index is given (e.g. `foo/bar`) -->
+move to that directory.
+
+See also issue #135.
 """
 function get_fs_path(req_path::AbstractString)::String
     uri     = HTTP.URI(req_path)
     r_parts = HTTP.URIs.unescapeuri.(split(lstrip(uri.path, '/'), '/'))
-    fs_path = joinpath(r_parts...)
+    fs_path = joinpath(CONTENT_DIR[], r_parts...)
 
-    if !isempty(CONTENT_DIR[])
-        fs_path = joinpath(CONTENT_DIR[], fs_path)
+    #
+    # Check if it's pointing to an explicit or implicit index file
+    #
+    append = (r_parts[end] == "index.html")
+    tmp = ifelse(
+        append,
+        fs_path,
+        joinpath(fs_path, "index.html")
+    )
```

Comment:
```suggestion
    tmp = append ? fs_path : joinpath(fs_path, "index.html")
```

Reply:
well the `ifelse` avoids a branch, it's completely irrelevant here as the gain is completely negligible but it is better practice to use `ifelse` in general if you can, so I'd rather leave it as is